seismicpro/src/survey.py

```diff
@@ -464,19 +473,19 @@ def load_first_breaks(self, path, first_breaks_col='FirstBreak'):
         Raises
         ------
         ValueError
-            If `FieldRecord` or `TraceNumber` headers were not loaded.
-            If ('FieldRecord', 'TraceNumber') pairs from the file do not match those in `self.headers`.
+            If not all the `trace_id_columns` are loaded in `self.headers`
+            If there is not a single match of rows from the file with those in `self.headers`.
         """
-        segy_columns = ['FieldRecord', 'TraceNumber']
-        first_breaks_columns = segy_columns + [first_breaks_col]
-        first_breaks_df = pd.read_csv(path, names=first_breaks_columns, delim_whitespace=True, decimal=',')
+        file_columns = trace_id_columns + (first_breaks_column, )
```

Comment:
feels bad about this line without a cast to any array-like.. for example
```suggestion
        file_columns = to_list(trace_id_columns) + [first_breaks_column]
```